articles/day1.md.html

```diff
@@ -4,38 +4,42 @@
+Keep in mind that everything we're going to show you in this course is only _one way_ of programming a game. In this course, we will present you how [Casey][] makes games, so that you can see one __complete__ way of doing it. You are encouraged however to develop your own styles and techniques, and experiment with trying different things and see what works the best for you.
 
-No prior knowledge of programming is required to start following along. The only thing that will help you along the way is the basic knowledge of Windows, the insides of a computer (what's a CPU, graphics card, Memory (RAM), etc. at a very general notion). Knowledge about that binary exists, that a byte is 8 bits and a Kibibyte (KiB) is 1024 bytes, and other basic computer theory, as well as the basics of color theory would definitely help as well.
+No prior knowledge of programming is required to start following along. The only thing that will help you along the way is the basic knowledge of Windows, the insides of a computer on a basic level (what's a CPU, Graphics card, Memory, Motherboard). Knowledge about that binary number system, that a byte is 8 bits and that a kilobyte is 1024 bytes, as well as the basics of color theory (RGB) would definitely help as well.
 
 # Start the Project
 
-The first thing we're going to do is to actually get the project started. We do it by setting up all our tools and by building _anything_ to ensure that the tools are working properly. While it may be a bit of _drudgework_, it's a vital step to ensure that, moving forward, the project can be continued successfully without any interruptions. 
+The first thing we're going to do is to get a simple project started. We do it by setting up all our tools and by building _something_ to ensure that our tools are working properly. While it may be a bit of _drudgework_, it's a vital step to ensure that when moving forward, the project can be continued successfully without any interruptions.
 
-At its very core, the following things must be created: 
+To program our game, for the starters we will only need the following tools:
 
-*   Something that allows you to build the actual text files that you type into the source ("a text editor")
-*   Something that allows you to build the text files into the executables ("compiler", "linker", etc).
+*   Text editor, that allows us to edit our code.
+*   Compiler, that allows us to compile our code and produce executable (.exe) files.
 
-You will also need a variety of additional tools that would render your life much simpler. All of these will be the focus of this Day.
+These tools, as well as few more advanced tools will be the focus of this day.
 
```

Comment:
The convention I adopted that each chapter is called "Day". One may take a whole week to go through a day, or a few hours, we don't know that. That's why Day is capitalized.